pkg/cmd/manager_controller.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"context"
+
+	"github.com/scylladb/go-log"
+	"github.com/scylladb/scylla-operator/pkg/cmd/options"
+	"github.com/scylladb/scylla-operator/pkg/controllers/manager"
+	"github.com/spf13/cobra"
+
+	"go.uber.org/zap"
```

Comment:
In scylla proper we normally use our own clones to avoid a left-pad like situation. The solution in here is to add the modules we use to vendor/, correct?

Reply:
Yep, vendor is solution for these kind of situations. We do have vendor dir in Operator, i forgot to update it.